### ISSUE TITLE
[v20] SelectButton - Focus & keyboard navigation completely broken

### ISSUE THREAD

**Issue by infe42** [primefaces/primeng#18596]
### Describe the bug

The SelectButton in v20 is no longer focusable and keyboard navigation doesn't work at all. This was all working in v19.

### Reason for not contributing a PR

- [ ] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/selectbutton

### Environment

-

### Angular version

20.0.6

### PrimeNG version

v20

### Browser(s)

Firefox

### Steps to reproduce the behavior

1. Go to https://primeng.org/selectbutton
2. Try to focus any SelectButton
3. Try to use keyboard navigation (arrows, tab, space, enter, ...)
--> not working

### Expected behavior

Should behave like https://v19.primeng.org/selectbutton
